website/pages/en/cookbook/_meta.js

```diff
@@ -11,4 +11,5 @@ export default {
   'immutable-entities-bytes-as-ids': 'Subgraph Best Practice 3: Using Immutable Entities and Bytes as IDs',
   'avoid-eth-calls': 'Subgraph Best Practice 4: Avoid eth_calls',
   'transfer-to-the-graph': '',
+  enums: 'Categorizing NFT Marketplaces Using Enums',
```

Comment:
@idalithb – Why customize the page title in the nav to make it so similar (just changing "Categorize" to "Categorizing")?